tests/integration/slowquery_test.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+
+package integration
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"gorm.io/gorm"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/slowquery"
+	"github.com/pingcap/tidb-dashboard/pkg/utils"
+	"github.com/pingcap/tidb-dashboard/util/testutil"
+)
+
+const (
+	TestSlowQueryTableName = "test.CLUSTER_SLOW_QUERY"
+)
+
+type testSlowQuerySuite struct {
+	suite.Suite
+	db               *testutil.TestDB
+	slowqueryColumns []string
```

Comment:
This part doesn't look good. It relies heavily on internal implementations. It should only occur in unit tests. The methodology of the integration test should be not relying on other internal implementation details at all. Otherwise it may be hard for others to write something similar for another module.